cmd/commands/function.go

```diff
@@ -46,51 +46,44 @@ func FunctionCreate(fcTool *core.Client) *cobra.Command {
 
 	// runtime definitions
 	buildpacks := map[string]string{
-		"java-buildpack": "projectriff/buildpack",
-		"detect":         "projectriff/buildpack",
+		"java": "projectriff/buildpack",
 	}
 	invokers := map[string]string{
-		"java":            "https://github.com/projectriff/java-function-invoker/raw/v0.1.1/java-invoker.yaml",
-		"java-invoker":    "https://github.com/projectriff/java-function-invoker/raw/v0.1.1/java-invoker.yaml",
-		"command":         "https://github.com/projectriff/command-function-invoker/raw/v0.0.7/command-invoker.yaml",
-		"command-invoker": "https://github.com/projectriff/command-function-invoker/raw/v0.0.7/command-invoker.yaml",
-		"node":            "https://github.com/projectriff/node-function-invoker/raw/v0.0.8/node-invoker.yaml",
-		"node-invoker":    "https://github.com/projectriff/node-function-invoker/raw/v0.0.8/node-invoker.yaml",
+		"jar":     "https://github.com/projectriff/java-function-invoker/raw/v0.1.1/java-invoker.yaml",
+		"command": "https://github.com/projectriff/command-function-invoker/raw/v0.0.7/command-invoker.yaml",
+		"node":    "https://github.com/projectriff/node-function-invoker/raw/v0.0.8/node-invoker.yaml",
 	}
 
 	command := &cobra.Command{
 		Use:   "create",
 		Short: "Create a new function resource",
 		Long: "Create a new function resource from the content of the provided Git repo/revision or local source.\n" +
-			"\nThe RUNTIME arg defines the language runtime that is added to the function code in the build step. The resulting image is then used to create a Knative Service (`service.serving.knative.dev`) instance of the name specified for the function. The following runtimes are available:\n\n" +
-			// TODO make runtime help data-driven
-			"- 'java': uses riff's java-function-invoker (aliased as java-invoker)\n" +
-			"- 'node': uses riff's node-function-invoker (aliased as node-invoker)\n" +
-			"- 'command': uses riff's command-function-invoker (aliased as command-invoker)\n" +
-			"- 'java-buildpack': uses the riff Buildpack \n" +
-			"- 'detect': uses the riff Buildpack's detection (currently limited to Java functions) \n" +
-			"\nClassic riff Invoker runtimes are available in addition to experimental Buildpack runtimes.\n" +
-			"\nBuildpack based runtimes support building from local source in addition to within the cluster. Locally built images prefixed with 'dev.local/' are saved to the local Docker daemon while all other images are pushed to the registry specified in the image name.\n" +
+			"\nThe INVOKER arg defines the language runtime that is added to the function code in the build step. The resulting image is then used to create a Knative Service (`service.serving.knative.dev`) instance of the name specified for the function. The following invokers are available:\n\n" +
```

Comment:
Could we say `The INVOKER arg defines the language runtime and function invoker that is added to the function code in the build step.`